### PR TITLE
docs/technical: Add snapshots documentation

### PR DESCRIPTION
**Pull request by solj**
This commit adds a section to the storage documentation describing the snapshots that are available for users to do self-directed restores.

Signed-off-by: Sol Jerome <sol.jerome@gmail.com>

### CODE REVIEW

File: docs/technical/storage.md [hunk 1]
```diff
@@ -4,11 +4,22 @@ Below is a table of all available storage on Unity.
...
+| `/scratch` | Scratch space       | Everywhere (Intended for Compute) | SSD      | 40 TB / user, cleared at the end of job | `/scratch/[nodeid]/[jobid]` is created when a job is started. That folder is assigned to $TMP and deleted after the job is complete. This directory is not directly available to users. |
+| `/gypsum`  | Gypsum devices      | Everywhere                        | HDD      | Varying                                 | For users migrating from the Gypsum cluster to the Unity clusters, you will find all your old storage here. |
+| `/old`     | Old mounts          | Everywhere (Read-Only)            | Varying  | Varying                                 | Old filesystems which are deprecated live here until they are deleted. |
+
+## Snapshots
+
+Backups are not available on the Unity cluster. There are temporary
+snapshots created each day at 1am UTC. Self-directed restores are
+accomplished by accessing read-only snapshots (see table below).
```
Comment (simonleary-umass-edu) [inline docs/technical/storage.md:20]:
```suggestion
Backups are not available on the Unity cluster.
There are temporary snapshots created each day at 1am UTC.
Snapshots older than three days are deleted.
Self-directed restores are accomplished by accessing read-only snapshots (see table below).
```

File: docs/technical/storage.md [hunk 2]
```diff
@@ -4,11 +4,22 @@ Below is a table of all available storage on Unity.
...
+
```
Comment (simonleary-umass-edu) [inline docs/technical/storage.md:21]: Perhaps an example of how to recover a snapshot:
```sh
rsync -r /snapshots/$HOME/ $HOME
```
(not 100% that command works as expected)

File: docs/technical/storage.md [hunk 3]
```diff
@@ -4,11 +4,22 @@ Below is a table of all available storage on Unity.
...
+| Filesystem | Name | Snapshot location |
+| --- | --- | --- |
+| `/home/<username>` | Home directory | `/snapshots/home/unity_<timestamp>/<username>` |
+| `/work/pi_<pi username>` | Work directory | `/snapshots/work/unity_<timestamp>/<pi_username>` |
```
Comment (simonleary-umass-edu) [inline docs/technical/storage.md:25]:
```suggestion
| `/work/pi_<pi-username>` | Work directory | `/snapshots/work/unity_<timestamp>/<pi-username>` |
```

**Review by simonleary-umass-edu — CHANGES_REQUESTED** [review]

File: docs/technical/storage.md (hunk 3 above)
Comment (solj) [reply docs/technical/storage.md:25]: > 

@simonleary-umass-edu as far as I can tell the path is `/work/pi_<username>` where username is the PI's username. I could change the first column to pi_username if you think that'd make it clearer.

Comment (simonleary-umass-edu) [reply docs/technical/storage.md:25]: I think the third column should match the first, `pi_<pi-username>`.

Comment (simonleary-umass-edu) [reply docs/technical/storage.md:25]: Whatever you choose, they should be the same on 1st and 3rd columns

File: docs/technical/storage.md (hunk 2 above)
Comment (solj) [reply docs/technical/storage.md:22]: I added an example. Let me know what you think of it.

File: docs/technical/storage.md (hunk 3 above)
Comment (solj) [reply docs/technical/storage.md]: This should be fixed now.

**Review by simonleary-umass-edu — APPROVED** [review]

**Comment by simonleary-umass-edu** [comment]
Could you squash these three commits into one commit?

**Pull request merged**